context/app/static/js/components/detailPage/SummaryItem.jsx

```diff
@@ -1,15 +1,6 @@
 import React from 'react';
 import styled from 'styled-components';
 import Typography from '@material-ui/core/Typography';
-import Divider from '@material-ui/core/Divider';
-
-const VerticalDivider = styled(Divider)`
-  margin-left: ${(props) => props.theme.spacing(1)}px;
-  margin-right: ${(props) => props.theme.spacing(1)}px;
-  height: 15px;
-  background-color: ${(props) => props.theme.palette.text.primary};
-  align-self: center;
-`;
```

Comment:
I think with the removal of the `VerticalDivider` this component has lost its use. The flex is only used to move the divider and icon in-line, so this is only really needed in situations where the `statusIcon` is passed.

Reply:
For most cases you can just use `Typography` now.